x/staking/spec/01_state.md

```diff
@@ -24,6 +24,7 @@ type Params struct {
     MaxEntries    uint16        // max entries for either unbonding delegation or redelegation (per pair/trio)
     BondDenom     string        // bondable coin denomination
 }
+// Thoughts: Params are being modified by governance, validator update should be done on next epoching based on Params change.
```

Comment:
Would be also nice to be able to define epoching intervals in terms of both block height or block time.